src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseAboutToStartHandlerTest.java

```diff
@@ -41,10 +41,13 @@
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsDocument;
 import uk.gov.hmcts.reform.sscs.ccd.domain.SscsDocumentDetails;
+import uk.gov.hmcts.reform.sscs.util.DynamicListLanguageUtil;
 
 @ExtendWith(MockitoExtension.class)
 class AdjournCaseAboutToStartHandlerTest {
 
+    @Mock
+    private DynamicListLanguageUtil dynamicListLanguageUtil;
```

Comment:
add a space between the mock and other global variables

Reply:
fixed